FirebaseVertexAI/Sources/Types/Public/Imagen/ImagenModel.swift

```diff
@@ -74,18 +79,25 @@ public final class ImagenModel {
     return try await generativeAIService.loadRequest(request: request)
   }
 
-  func imageGenerationParameters(storageURI: String?) -> ImageGenerationParameters {
-    // TODO(#14221): Add support for configuring these parameters.
+  func imageGenerationParameters(storageURI: String?,
```

Comment:
I'm missing a good bit of context here, but when/why would the `storageURI` parameter be optional? I would think that each image gen would map to a URI.

Reply:
Images can be returned in two different ways:
1. Inline in the response payload as a base64-encoded string (default behaviour).
2. Saved in a GCS bucket (a subdirectory of `storageURI`).

Sorry it doesn't help that there isn't any code documentation yet to explain that.

Reply:
Aha, makes sense, thanks!